tensorflow-proto/tensorflow-proto.cabal

```diff
@@ -40,6 +40,10 @@ library
   default-language:    Haskell2010
   include-dirs: .
 
+custom-setup
+  setup-depends: Cabal
+                , proto-lens-protoc == 0.2.*
```

Comment:
Oh, right, `proto-lens-protoc` as a *library* is also providing `Data.ProtoLens.Reexport.*`.

I'll take a look at whether there's a better scheme for organizing our packages now that `stack` works better with `setup-depends`.